src/test/java/lotto/domain/LottoCountTest.java

```diff
@@ -0,0 +1,43 @@
+package lotto.domain;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class LottoCountTest {
+    private LottoCount lottoCount;
+
+    @BeforeEach
+    public void setUp() {
+        this.lottoCount = new LottoCount(new LottoPrice(14000), 3);
+    }
```

Comment:
```suggestion
    private final LottoCount lottoCount = new LottoCount(new LottoPrice(14000), 3);
```
junit에서 기본 설정값은 테스트메소드마다 클래스를 초기화하여 사용하기 때문에 setUp이 아닌 불변으로도 사용해볼 수 있습니다 😄 

https://junit.org/junit5/docs/current/user-guide/#writing-tests-test-instance-lifecycle